impeller/entity/geometry.cc

```diff
@@ -53,10 +53,13 @@ GeometryResult FillPathGeometry::GetPositionBuffer(
     const ContentContext& renderer,
     const Entity& entity,
     RenderPass& pass) {
+  auto tolerance =
+      kDefaultCurveTolerance / entity.GetTransformation().GetMaxBasisLength();
```

Comment:
Does the max basis length need to be bottom-bound to e.g. `0.1f` for this? What happens when the transform is scaled to 0 and the tolerance becomes `NaN`?

Reply:
Hello @bdero 
It becomes below when I set `auto tolerance = kDefaultCurveTolerance / 0;`
![image](https://user-images.githubusercontent.com/49340347/210480357-18381538-be57-47e4-a987-c8ac14a7bc5d.png)

And this change is almost the same like code in 
https://github.com/flutter/engine/blob/main/impeller/entity/geometry.cc#L420-#L422

Reply:
Thanks for checking! This seems like good behavior.